modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -203,8 +204,46 @@
             (is (= [{:name "numeric_col",   :database_type "numeric(10,2)",         :base_type :type/Decimal}
                     {:name "weird_varchar", :database_type "character varying(50)", :base_type :type/Text}]
                    (map
-                    (partial into {})
-                    (db/select [Field :name :database_type :base_type] :table_id table-id {:order-by [:name]}))))))))))
+                    mt/derecordize
+                    (db/select [Field :name :database_type :base_type] :table_id table-id {:order-by [:name]})))))
+          (finally
+            (redshift.test/execute! (str "DROP TABLE IF EXISTS %s;%n"
+                                         "DROP VIEW IF EXISTS %s;")
+                                    qual-tbl-nm
+                                    qual-view-nm))))))))
```

Comment:
Added this because we weren't cleaning up these tables/views before as far as I can tell...

Reply:
Nice catch